tests/testapp/migrations/0002_blogstreampage.py

```diff
@@ -40,7 +40,7 @@ class Migration(migrations.Migration):
                             (
                                 "heading",
                                 wagtail.core.blocks.CharBlock(
-                                    form_classname="full title", icon="title"
+                                    form_classname="title", icon="title"
```

Comment:
Edited to save us a migration but technically we’re already missing a migration for the introduction of `use_json_field`.